packages/11ty/_plugins/figures/test/__fixtures__/iiif-config.json

```diff
@@ -2,9 +2,9 @@
   "baseURI": "http://localhost:8080",
   "dirs": {
     "imagesDir": "_assets/images",
-    "inputRoot": "/Users/apollack/Code/getty/sequence-blarg/content",
+    "inputRoot": "/test-project/_site",
     "outputPath": "iiif",
-    "outputRoot": "/Users/apollack/Code/getty/sequence-blarg/_site"
+    "outputRoot": "/test-project/_site"
```

Comment:
the `inputRoot` and `outputRoot` are not actually being used as these generated figures are coming from fixture data, but it seems silly to have a fake path on my personal machine in this JSON...